IFC4x3/Properties/s/Slaking_1GTIyqmf9Eu8cGgYc0$TFp/DocProperty.xml

```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocProperty xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="Slaking_1GTIyqmf9Eu8cGgYc0TFp" Name="Slaking" UniqueId="50752f34-c292-4ee0-8990-aa2980fdd3f3" PrimaryDataType="IfcLabel" />
```

Comment:
`IfcLabel`?

Reply:
> `IfcLabel`?

According to excel spec.

Reply:
Okay, in that case this is resolved, right?

Reply:
IfcLabel it is.